sdk/python/jobs/pipelines/pipeline_with_components_use_primitive_output/pipeline_with_components_use_primitive_output.ipynb

```diff
@@ -0,0 +1,364 @@
+    "# set pipeline level compute\n",
+    "pipeline_job.settings.default_compute = \"cpu-cluster\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inspect built pipeline\n",
+    "print(pipeline_job)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.3 Submit pipeline job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit pipeline job to workspace\n",
+    "pipeline_job = ml_client.jobs.create_or_update(\n",
+    "    pipeline_job, experiment_name=\"pipeline_samples\"\n",
+    ")\n",
+    "\n",
+    "pipeline_job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wait until the job completes\n",
+    "ml_client.jobs.stream(pipeline_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Define and Load components by `@command_component`\n",
+    "### 3.1 Define component function, and define primitive output in function\n",
+    "(`@pipeline`, `@dynamic` and `@command_component` are used the same way when using primary output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.1.1 Define a group primary outputs using `@group`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please review the definition of the \"component_return_annotated_group_outputs\" function\n",
+    "from components_with_annotated import component_return_annotated_group_outputs\n",
+    "print(component_return_annotated_group_outputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.1.2 Define a primary output\n",
+    "(`int`, `float`, `string`, `bool` are used the same way when define a primary output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please review the definition of the \"component_return_annotated_output\", \"component_return_int_output\", \"component_return_integer_output\" function\n",
+    "from components_with_annotated import component_return_annotated_output, component_return_int_output, component_return_integer_output\n",
+    "\n",
+    "print(component_return_annotated_output)\n",
+    "print(component_return_int_output)\n",
+    "print(component_return_integer_output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### 3.2 Build pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@pipeline()\n",
+    "def pipeline2_with_components_use_primitive_output():\n",
```

Comment:
we should avoid using number in function name

Reply:
Based on the discussion, delete these codes and modify existing codes